docs/src/userguide/interpolation_and_regridding.rst

```diff
@@ -417,22 +417,22 @@ In each case ``result`` will be the input cube regridded to the grid defined by
 the target grid cube (in this case ``rotated_psl``) that we used to define the
 cached regridder.
 
-Regridding Lazy Data
-^^^^^^^^^^^^^^^^^^^^
+Interpolating and Regridding Lazy Data
+--------------------------------------
 
-If you are working with large cubes, especially when you are regridding to a
-high resolution target grid, you may run out of memory when trying to
-regrid a cube. When this happens, make sure the input cube has lazy data
+If you are working with large cubes, you may run out of memory when trying to
+interpolate or regrid a cube. For instance, this might happen when regridding to a
+high resolution target grid. When this happens, make sure the input cube has lazy data
 
     >>> air_temp = iris.load_cube(iris.sample_data_path('A1B_north_america.nc'))
     >>> air_temp
     <iris 'Cube' of air_temperature / (K) (time: 240; latitude: 37; longitude: 49)>
     >>> air_temp.has_lazy_data()
     True
 
-and the regridding scheme supports lazy data. All regridding schemes described
-here support lazy data. If you still run out of memory even while using lazy
-data, inspect the
+and the interpolation or regridding scheme supports lazy data. All interpolation and
+regridding schemes described here with exception of the the point-in-cell regridder
```

Comment:
The `CurvilinearRegridder` [used by the point-in-cell regridder](https://github.com/SciTools/iris/blob/67d0b0771834510b782ca57af9e445f6e2e21427/lib/iris/analysis/__init__.py#L3101) says it does not support lazy data: 
https://github.com/SciTools/iris/blob/67d0b0771834510b782ca57af9e445f6e2e21427/lib/iris/analysis/_regrid.py#L453-L454